ui/components-react/src/components-react/newEditors/editors/boolean/UseBooleanEditorProps.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { EditorProps, RequiredProps } from "../../Types.js";
+import type { BooleanValue, Value } from "../../values/Values.js";
+import { isBooleanValue } from "../../values/Values.js";
+
+/**
+ * Hooks that converts generic `EditorProps` into editor props with boolean value. If value is not boolean, it will be converted into `false`.
```

Comment:
> If value is not boolean, it will be converted into `false`.

Maybe we should use truthiness instead? E.g. `0` would convert to `false`, but any other number would convert to `true`.

Reply:
Removed conversion itself.